src/main/resources/application.yml

```diff
@@ -4,7 +4,18 @@ spring:
       database: UrlEntity
       host: localhost
       port: 27017
-
+  datasource:
+    username: root
+    password: admin
+    url: jdbc:mysql://localhost:3306/shortenUrl?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    initialization-mode: always
+  jpa:
+    database-platform: org.hibernate.dialect.MySQL5InnoDBDialect
+    generate-ddl: true
+    show-sql: true
+    hibernate:
+      ddl-auto: update
```

Comment:
Should it be update?

Reply:
I checked and the initilization works if it is none